extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -536,7 +537,17 @@ export class AmpSlideScroll extends BaseSlides {
       this.updateInViewport(this.slides_[
           dev().assertNumber(this.slideIndex_)], false);
     }
-    this.updateInViewport(this.slides_[newIndex], true);
+    const newSlideInView = this.slides_[newIndex];
+
+    if (newSlideInView === undefined) {
+      dev.error(
+        TAG,
+        'Accessing a non-existant slide at index: %s, noOfSlides: %s',
```

Comment:
Sorry about that i blindly converted from one method to another , fixed it now - also added a return;  (since we are logging this.)